platformcomponents/desktop/date-picker.json

```diff
@@ -39,22 +39,45 @@
     },
     "dayPicker":{
       "day":{
-        "background":{
-          "#normal": "@theme-button-secondary-normal",
-          "#hovered": "@theme-button-secondary-hover",
-          "#pressed": "@theme-button-secondary-pressed",
-          "#disabled": "@theme-button-secondary-normal",
-          "#active": "@theme-background-accent-normal"
+        "unchecked": {
+          "#normal": {
+            "background": "@theme-button-secondary-normal",
+            "text": "@theme-text-primary-normal",
+            "border": "@theme-outline-button-normal"
+          },
+          "#hovered": {
+            "background": "@theme-button-secondary-hover",
+            "text": "@theme-text-primary-normal",
+            "border": "@theme-outline-button-normal"
+          },
+          "#pressed": {
+            "background": "@theme-button-secondary-pressed",
+            "text": "@theme-text-primary-normal",
+            "border": "@theme-outline-button-normal"
+          },
+          "#disabled": {
+            "background": "@theme-button-secondary-normal",
+            "text": "@theme-text-primary-disabled",
+            "border": "@theme-outline-button-normal"
+          }
         },
-        "text": {
-          "#normal": "@theme-text-primary-normal",
-          "#hovered": "@theme-text-primary-normal",
-          "#pressed": "@theme-text-primary-normal",
-          "#disabled": "@theme-text-primary-disabled",
-          "#active": "@theme-common-text-white"
-        },
-        "border": {
-          "#normal": "@theme-outline-button-normal"
+        "checked": {
+          "#normal": {
+            "background": "@theme-background-accent-normal",
+            "text": "@theme-common-text-white"
+          },
+          "#hovered": {
+            "background": "@theme-background-accent-hover",
+            "text": "@theme-common-text-white"
+          },
+          "#pressed": {
+            "background": "@theme-background-accent-pressed",
```

Comment:
I see theme-button-accent-pressed in the figma.. theme-background-accent-pressed doesn't exist in theme tokens.